app/models/ncr.rb

```diff
@@ -0,0 +1,9 @@
+module Ncr
+  # Make sure all table names use 'ncr_XXX'
+  def self.table_name_prefix
+    'ncr_'
+  end
+
+  EXPENSE_TYPES = %w(BA60 BA61 BA80)
```

Comment:
wondering why these are in the module and not the class itself? (I know this isn't a change, but I've thought this every time I visited `work_order` so thought I'd ask)